### PR TITLE
Switch from bower to npm

### PR DESCRIPTION
**Pull request by jzaefferer**
Fixes the broken bower dependencies, highlight.js can't be installed anymore.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,23 +6,10 @@ Internationalization is the process that provides users with a localized experie
...
-Install the [bower](http://bower.io/) packages if you haven't before. These should be done as global installs:
-
-```bash
-npm install -g bower # once
-```
-
-Then install the dependencies (reveal.js and highlight.js) by running bower:
-
-```
 bower install
```
Comment (rxaviers) [inline README.md]: `s/bower/npm`?

Comment (jzaefferer) [inline README.md]: yep, fixed

**Comment by rxaviers** [comment]
Thanks!

**Comment by rxaviers** [comment]
Closed by 79cf9d4ece17913fe912485f16babbd737b8339d

**Pull request closed without merging**